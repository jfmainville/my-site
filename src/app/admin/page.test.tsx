import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import AdminPage from "./page";

describe("<AdminPage/> Test Suite", () => {
  it("should render the <AdminPage/> page unchanged", () => {
    const { container } = render(<AdminPage />);
    expect(container).toMatchSnapshot();
  });
});
