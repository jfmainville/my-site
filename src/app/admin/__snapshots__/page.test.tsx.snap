// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminPage/> Test Suite should render the <AdminPage/> page unchanged 1`] = `
<div>
  <section
    class="Main"
  >
    <div
      class="Navbar"
    >
      <div>
        Admin
      </div>
      <div>
        <button
          type="submit"
        >
          <a
            href="/api/auth/logout"
          >
            Logout
          </a>
        </button>
      </div>
    </div>
    <section
      class="Section"
    >
      <a
        class="SectionLinkItem"
        href="/admin/post"
      >
        <div>
          Blog Posts
        </div>
      </a>
    </section>
  </section>
</div>
`;
